sar-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>net.kaczmarzyk</groupId>
+    <artifactId>sar-spring-boot-starter</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Autoconfiguration pom has different version than main pom, but in order to keep the same version for both I would have to create another "core" module and move whole main code base to it. In other case I can't add it as a dependency in this pom. Should I do it?

Reply:
https://stackoverflow.com/questions/10024320/maven-include-parent-classes <--- additional info on solution to this problem